core/test/nuxeo-operation.test.js

```diff
@@ -34,6 +34,49 @@ suite('nuxeo-operation', () => {
     server.restore();
   });
 
+  function getNuxeoPageProvider(hasExcludeDocs = false) {
+    if (hasExcludeDocs) {
+      return fixture(html`
+        <nuxeo-page-provider
+          id="nx-pp"
+          provider="test_provider"
+          page="2"
+          page-size="40"
+          sort='{"field": "asc"}'
+          params='{
+          "boolean": false,
+          "excludeDocs": ["fabf0fa3-0f0a-4b26-8fde-9c4ac869cd5f","f17544e4-5945-4b5c-b4a9-d48d2468b75b"]
+        }'
+        ></nuxeo-page-provider>
+      `);
+    }
+    return fixture(html`
+      <nuxeo-page-provider
+        id="nx-pp"
+        provider="test_provider"
+        page="2"
+        page-size="40"
+        sort='{"field": "asc"}'
+        params='{"boolean": false}'
+      ></nuxeo-page-provider>
+    `);
+  }
+
+  function getNuxeoOperation(provider) {
+    if (typeof provider !== 'undefined') {
```

Comment:
nit: since we are dealing with a function parameter, I don't think this offers any benefit over the shorter:
```suggestion
    if (provider != null) {
```